front/src/Phaser/Map/ITiledMap.ts

```diff
@@ -167,14 +167,21 @@ export interface ITiledTileSet {
     tilewidth: number;
     transparentcolor: string;
     terrains: ITiledMapTerrain[];
-    tiles: {[key: string]: { terrain: number[] }};
+    tiles?: Array<ITile>;
```

Comment:
🤔 the previous type doesnt make sense to me at all 😅

Reply:
:+1: